### DISCUSSION TITLE
I can't upload my student certificate.

### DISCUSSION THREAD

**Discussion by seliminsan** [community/community#76945] [GitHub Education]
### Hi everyone,

There is no option to upload student certificate.

![bug](https://github.com/community/community/assets/117529414/57502b01-0aec-46e1-bfe8-dcc885d09a7e)

**Comment by nurahmed123** [comment]


**Reply by seliminsan** [reply]
I tried your solution but it still doesn't work. I will print out my student ID and take a photo of it with my phone and send it.
